nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -1006,7 +1005,7 @@ def build(self, target_model: NNCFNetwork) -> ElasticWidthHandler:
             nodes = graph.get_nodes_by_metatypes([metatype])
             for node in nodes:
                 node_name = node.node_name
-                nncf_logger.info("Adding Dynamic Input Op for {} in scope: {}".format(metatype.name, node_name))
+                nncf_logger.info(f"Adding Dynamic Input Op for {metatype.name} in scope: {node_name}")
```

Comment:
Propose moving to debug
@jpablomch

Reply:
@ljaljushkin Moving to debug works for us. Thanks!

Reply:
Done